src/DSE.Open.Results/ResultHttpStatusCodeMapper.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Down Syndrome Education International and Contributors. All Rights Reserved.
+// Down Syndrome Education International and Contributors licence this file to you under the MIT license.
+
+using System.Net;
+
+namespace DSE.Open.Results;
+
+public static class ResultHttpStatusCodeMapper
+{
+    public static HttpStatusCode ToHttpStatusCode(this ResultStatus status)
+    {
+        return status switch
```

Comment:
Unless we are going to start returning the most correct verbs, I'm not sure we need most of these. Probably just BadRequest, NotFound, Unauthorized, and Forbidden?

Reply:
We should review this with respect to the APIs